playground-sveltekit/vite.config.js

```diff
@@ -1,9 +1,9 @@
 import { sveltekit } from "@sveltejs/kit/vite";
-import mkcert from "vite-plugin-mkcert";
+import basicSsl from "@vitejs/plugin-basic-ssl";
```

Comment:
Interesting to change this. Curious - is there an issue with the mkcert plugin or is the basic-ssl plugin favourable for a particular reason?

Reply:
> Interesting to change this. Curious - is there an issue with the mkcert plugin or is the basic-ssl plugin favourable for a particular reason?

Hi @darbymanning  thank you for all your effort and also the feedback.
Let me reply to this question because i pushed to use basic-ssl-plugin.
This plugin is suggested in the Vite official doc : https://vitejs.dev/config/server-options.html#server-https
This plugin auto-generates a self-signed certificate out of the box. Instead, with mkcert you have to generate a certificate with the mkcert command line tool, and then execute a proxy.
So I think that with plugin-basic-ssl the setup is easier for new users.
This is just for the playground, a basic example so that you can use the best tool.
I am happy to know what you think.

Reply:
Sounds good to me! Thanks for taking the time to explain the rationale :)